.github/workflows/loristest.yml

```diff
@@ -0,0 +1,92 @@
+name: LORIS Test Suite
+
+on:
+  - push
+  - pull_request
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    strategy:
+        fail-fast: false
+        matrix:
+            testsuite: ['integration']
+            php: ['8.1','8.2', '8.3']
+            ci_node_index: [0]
+
+            include:
+            # add a variable but do not display it in the job's name
+            - ci_node_total: 1
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Setup PHP
+      uses: shivammathur/setup-php@v2
+      with:
+        php-version: ${{ matrix.php }}
+        extensions: zip, php-ast
+
+
+    - name: Clone Loris repo
+      run: git clone https://github.com/aces/Loris.git
+
+    - name: Override Test Files
+      run: |
+        cp test/Dockerfile.test.php8 Loris/
+        cp test/docker-compose.yml Loris/
+        cp test/phpunit.xml Loris/test/
+        cp test/RB_SQL/* Loris/raisinbread/RB_files/
+
+    - name: Validate composer.json and composer.lock
+      run: |
+        cd Loris
+        composer validate
+
+    - name: Cache Composer packages
+      id: composer-cache
+      run: |
+        cd Loris
+        echo "::set-output name=dir::$(composer config cache-files-dir)"
+
+    - name: Composer cache
+      uses: actions/cache@v3
+      with:
+        path: ${{ steps.composer-cache.outputs.dir }}
+        key: ${{ runner.os }}-composer-${{ hashFiles('Loris/composer.lock') }}
+        restore-keys: ${{ runner.os }}-composer-
+
+    - name: Change PHP Version in Dockerfile
+      run: |
+        cd Loris
+        sed -i "s/8.0/${{ matrix.php }}/g" Dockerfile.test.php8
+
+    - name: Install package dependencies
+      run: |
+        sudo apt install -y imagemagick-6-common libmagickcore-6.q16-6 libmagickwand-6.q16-6 \
+                            libprotobuf-dev libprotobuf23 libprotoc23 protobuf-compiler
+        cd Loris/modules/electrophysiology_browser/jsx/react-series-data-viewer/
+        protoc protocol-buffers/chunk.proto --js_out=import_style=commonjs,binary:./src/
+
+    - name: Install composer dependencies
+      if: steps.composer-cache.outputs.cache-hit != 'true'
+      run: |
+        cd Loris
+        composer install --prefer-dist --no-progress --no-suggest
+
+    - name: Build LORIS
+      run: |
+        cd Loris
+        make dev
+
+    - name: Run Test Suite
+      run: |
+          # set sandbox to 1 before running the tests
+          # since some tests only run in sandbox environments
+          sed -i 's/<sandbox>0<\/sandbox>/<sandbox>1<\/sandbox>/g' Loris/test/config.xml
+          cd Loris
```

Comment:
Why do we run the LORIS tests?

Reply:
@kongtiaowang? See Laetitia's question above. I don't understand either why we would need to run the LORIS tests here. Thank you :)

Reply:
yes, we need to install Loris first, it will include the all tables from raisinbread data. like config ... , we don't have python testsuit yet on gitactions, right?

Reply:
all good. Thanks!